src/platforms/rust/guides/actix-web/index.mdx

```diff
@@ -1,37 +1,26 @@
 ---
-draft: false
-categories: []
-toc: true
 title: actix-web
-sidebar_order: 2
-tags: []
+redirect_from:
+  - /platforms/rust/actix/
 ---
 
-_Crate: `sentry-actix` (has to be installed separately)_
+The `sentry-actix` crate adds a middleware for [`actix-web`](https://actix.rs/) that captures errors and report them to `Sentry`.
 
-The `sentry-actix` _crate_ adds a middleware for
-[`actix-web`](https://actix.rs/) that captures errors and report them to
-`Sentry`.
-
-To use this middleware, just configure Sentry and then add it to your actix web
-app as a middleware. Because actix is generally working with non sendable
-objects and is highly concurrent, this middleware creates a new hub per request.
-As a result many of the sentry integrations such as breadcrumbs do not work
-unless you bind the actix hub.
+To use this middleware, just configure Sentry and then add it to your actix web app as a middleware. Because actix is generally working with non sendable objects and is highly concurrent, this middleware creates a new hub per request. As a result many of the sentry integrations such as breadcrumbs do not work unless you bind the actix hub.
```

Comment:
```suggestion
To use this middleware, configure Sentry then add it to your actix web app as a middleware. Because actix is generally working with non sendable objects and is highly concurrent, this middleware creates a new hub per request. As a result, many of the Sentry integrations, such as breadcrumbs, do not work unless you bind the actix hub.
```